ga4gh/cli.py

```diff
@@ -357,10 +361,18 @@ def __init__(self, args):
         super(SearchVariantsRunner, self).__init__(args)
         request = RequestFactory(args).createSearchVariantsRequest()
         # if no variantSets have been specified, send a request to
-        # the server to grab all variantSets and then continue
+        # the server to grab all variantSets in the first dataset
```

Comment:
I don't see the distinction here @dcolligan --- if we leave the `variantSets` option unset, we're saying search over all variant sets. To me, this means
```python
for datasetId in getDatasets():
    for variantSet in getVariantSets(datasetId):
         # do stuff for the variant set
```